### PR TITLE
Fix deployment issues on lsst-lsp-int-dax

### PR DESCRIPTION
**Pull request by kennywlo**
Pls review.

### CODE REVIEW

File: kube/int/create.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -ex
-DAX_NAMESPACE=${DAX_NAMESPACE:-'dax-int'}
+DAX_NAMESPACE=${DAX_NAMESPACE:-'lsst-lsp-int-dax'}
 
 # kubectl create -f dax-metaserv-datasets-volume.yaml
```
Comment (cbanek) [inline kube/int/create.sh]: Please get rid of commented lines here and PVs.  Metaserv doesn't need them, and will never need them.

File: kube/int/dax-metaserv-deployment.yaml [hunk 2]
```diff
@@ -33,7 +33,7 @@ spec:
       volumes:
       - name: config
         secret:
-          secretName: dax-webserv-config
+          secretName: dax-metaserv-config
       - name: datasets
         persistentVolumeClaim:
-          claimName: dax-webserv-datasets-claim
```
Comment (cbanek) [inline kube/int/dax-metaserv-deployment.yaml:39]: Get rid of this, you don't need it.

**Review by cbanek — CHANGES_REQUESTED** [review]

File: kube/int/create.sh (hunk 1 above)
Comment (kennywlo) [reply kube/int/create.sh]: Done.

**Review by cbanek — CHANGES_REQUESTED** [review]
You missed destroy.sh, get rid of commented lines.

File: kube/int/dax-metaserv-deployment.yaml (hunk 2 above)
Comment (kennywlo) [reply kube/int/dax-metaserv-deployment.yaml:39]: Should be all fixed.

**Comment by kennywlo** [comment]
Good to go?

**Review by cbanek — APPROVED** [review]

**Pull request merged**